web_sdk/sdk_rewriter.dart

```diff
@@ -79,6 +79,7 @@ List<Replacer> generatePartsPatterns(String libraryName, bool isPublic) {
     // Remove library directives.
     AllReplacer(RegExp(r'\nlibrary .+;'), ''),
     // Remove imports/exports from all part files.
+    AllReplacer(RegExp(r"\nimport '.+'\n\s*if \(dart\.library\..*\) '.*';"), ''),
```

Comment:
Not that it matters too much, but I'd do a simpler regex to handle both `import` cases?

https://regexr.com/8a1l1

?